api/pkg/s3/datatype/datetime.go

```diff
@@ -0,0 +1,31 @@
+package datatype
```

Comment:
Please add the license announcement at the begining

Reply:
ok, thanks for your suggestion, will add the license in the code.